snapshot/lib/snapshot/simulator_launchers/simulator_launcher.rb

```diff
@@ -0,0 +1,74 @@
+require 'snapshot/simulator_launchers/simulator_launcher_base'
+
+module Snapshot
+  class SimulatorLauncher < SimulatorLauncherBase
+    def default_number_of_simultaneous_simulators
+      cpu_count = OS.cpu_count
```

Comment:
This will need some comments, I can imagine why we do this, but it would be good to have it documented here